rules/linux/builtin/lnx_privileged_user_creation.yml

```
@@ -0,0 +1,33 @@
+title: Creation of Privileged User
+id: 0ac15ec3-d24f-4246-aa2a-3077bb1cf90e
+status: experimental
+description: Detects when privileged user is added to the environment. 
+#The example of the events that could be observed when matching these would be as follow 
+#Dec 21 16:42:19 testserver useradd[1337]: new user: name=butter1, UID=1000, GID=0, home=/root, shell=/bin/bash
+#Dec 21 17:13:54 testserver useradd[1337]: new user: name=john, UID=0, GID=0, home=/home/john, shell=/bin/bash
+#Dec 21 17:24:40 testserver useradd[1337]: new user: name=butter3, UID=1000, GID=10, home=/home/butter3, shell=/bin/bash
+#Dec 21 17:30:22 testserver useradd[1337]: new user: name=butter4, UID=1000, GID=27, home=/home/butter4, shell=/bin/bash
+references:
+    - https://digital.nhs.uk/cyber-alerts/2018/cc-2825
+    - https://linux.die.net/man/8/useradd
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1136.001/T1136.001.md#atomic-test-5---create-a-new-user-in-linux-with-root-uid-and-gid
+author: Pawel Mazur
+date: 2022/12/21
+tags:
+    - attack.t1136.001
+    - attack.t1098
+    - attack.persistence
+logsource:
+    product: linux
+detection:
+    selection_new_user|contains:
+        - 'new user'
+    selection_uids_gids|contains:
```

```suggestion
    selection_uids_gids:
```
eg https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#lists